app/presenters/umrdr/work_show_presenter.rb

```diff
@@ -27,7 +27,14 @@ def grantnumber
       
     def doi
       @solr_document[Solrizer.solr_name('doi', :symbol)].first
-      
+    end
+
+    def tombstone
+      if @solr_document[Solrizer.solr_name('tombstone', :symbol)].nil?
+        nil
+      else
+        @solr_document[Solrizer.solr_name('tombstone', :symbol)].first
+      end
```

Comment:
@solr_document[Solrizer.solr_name('tombstone', :symbol)]&.first

I think this will work instead, ask bill :)